template/flake.nix

```diff
@@ -2,80 +2,92 @@
   description = "Soxin template flake";
 
   inputs = {
-    nixos.url = "nixpkgs/nixos-20.09";
-    master.url = "nixpkgs/master";
+    deploy-rs.url = github:serokell/deploy-rs;
+    nixpkgs.url = github:NixOS/nixpkgs/release-21.05;
+    nur.url = github:nix-community/NUR;
+    sops-nix.url = github:Mic92/sops-nix;
+    unstable.url = github:NixOS/nixpkgs/nixos-unstable;
+    utils.url = github:gytis-ivaskevicius/flake-utils-plus/v1.1.0;
+
     home-manager = {
-      url = "github:nix-community/home-manager/release-20.09";
-      inputs.nixpkgs.follows = "nixos";
+      url = "github:nix-community/home-manager";
+      inputs.nixpkgs.follows = "nixpkgs";
     };
+
     soxin = {
-      url = "github:SoxinOS/soxin";
+      url = path:../.;
```

Comment:
This won't work when initializing the template with `nix flake init -t github:SoxinOS/soxin`

Reply:
True and my intention is to put it back and remove the lock file before we merge.

Reply:
It's done.